packages/flutter/test/material/tabs_test.dart

```diff
@@ -4309,7 +4444,8 @@ void main() {
     expect(pageView.physics.toString().contains('ClampingScrollPhysics'), isFalse);
   });
 
-  testWidgets('TabController changes offset attribute', (WidgetTester tester) async {
+  testWidgets('TabController.offset changes reflect labelColor',
+          (WidgetTester tester) async {
```

Comment:
no need for a newline here

Reply:
it was passing the 80 character limit, that's why I did it. will revert it sure.